task/sbom-json-check/0.1/sbom-json-check.yaml

```diff
@@ -16,6 +16,8 @@ spec:
   results:
     - description: Tekton task test output.
       name: TEST_OUTPUT
+    - description: Images processed in the task.
+      name: IMAGES_PROCESSED
   steps:
   - name: sbom-json-check
     image: quay.io/redhat-appstudio/hacbs-test:v1.3.0@sha256:cd4601a7d71ebd908046db7a9b7010611b8b372fe941664d5163c81250a1a1fc
```

Comment:
needs newer image to work

Reply:
yes, was waiting for a new version to release. Thanks for merging that PR. Updated the image here